app/src/org/commcare/navdrawer/BaseDrawerActivity.kt

```
@@ -0,0 +1,256 @@
+        val profilePic = user.photo
+        if (!profilePic.isNullOrEmpty()) {
+            try {
+                val base64Part = profilePic.substringAfter("base64,", profilePic)
+                val decodedBytes = Base64.decode(base64Part, Base64.DEFAULT)
+                val bitmap = BitmapFactory.decodeByteArray(decodedBytes, 0, decodedBytes.size)
+                if (bitmap != null) {
+                    userPhoto.setImageBitmap(bitmap)
+                } else {
+                    userPhoto.setImageResource(R.drawable.nav_drawer_person_avatar)
+                }
+            } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)